warehouse/query-core/src/main/java/datawave/query/jexl/DefaultArithmetic.java

```diff
@@ -178,7 +179,7 @@ public boolean lessThan(Object left, Object right) {
             Set<Object> set = (Set<Object>) left;
 
             for (Object o : set) {
-                if (super.compare(o, right, LESS_THAN) < 0) {
+                if (super.compare(o, right, JexlOperator.LT) < 0) {
```

Comment:
I don't think the string constants on line 16 are needed anymore

Reply:
+1

Reply:
Still not resolved